internal/api/v1beta1/role.go

```diff
@@ -201,8 +244,31 @@ func (h Handler) UpdateOrganizationRole(ctx context.Context, request *shieldv1be
 	return &shieldv1beta1.UpdateOrganizationRoleResponse{Role: &rolePB}, nil
 }
 
+func (h Handler) DeleteRole(ctx context.Context, request *shieldv1beta1.DeleteRoleRequest) (*shieldv1beta1.DeleteRoleResponse, error) {
```

Comment:
protect this endpoint

Reply:
same as above, was already there in the interceptor